server/views/admin/licences/activity.pug

```diff
@@ -19,10 +19,18 @@ mixin send(details)
           | Sent from CA → RO
       when 'roToCa'
           | Sent from RO → CA
+      when 'roToCaAddressRejected'
+          | Sent from RO → CA (Address rejected)
+      when 'roToCaOptedOut'
+          | Sent from RO → CA (Opted out)
       when 'caToDm'
           | Sent from CA → DM
+      when 'caToDmRefusal'
+          | Sent from CA → DM (Refusal)
       when 'dmToCa'
           | Sent from DM → CA
+      when 'dmToCaReturn'
+          | Sent from DM → CA (Return)
       default
```

Comment:
These weren't mapped so events with these transitions appeared blank in the admin audit history screen